backend/core/src/main/java/com/knewless/core/elasticsearch/dto/RangeBoundaryType.java

```diff
@@ -0,0 +1,40 @@
+package com.knewless.core.elasticsearch.dto;
+
+import lombok.extern.slf4j.Slf4j;
+import org.elasticsearch.index.query.RangeQueryBuilder;
+
+@Slf4j
+public enum RangeBoundaryType {
+	/**
+	 * Greater than '>'
+	 */
+	GT,
+	/**
+	 * Greater than or equals '>='
+	 */
+	GTE,
+	/**
+	 * Less than '<'
+	 */
+	LT,
+	/**
+	 * Less than or equals '<='
+	 */
+	LTE;
```

Comment:
possible to make one-line comments?

Reply:
No. It's not displayed as javadoc then. 
![example](https://user-images.githubusercontent.com/30311108/91433820-84c77e80-e86c-11ea-851b-27bc4a187055.png)